Add the Aishell2 recipe for the master branch.

pengchengguo opened the pull request:
Add the Aishell2 recipe for the master branch.

sw005320 commented on egs/aishell2/asr1/conf/train.yaml:
```diff
@@ -0,0 +1,40 @@
+# network architecture
```
- Same comment: You can use a symlink for this and specify the actual config in the tuning directory. Please check the other recipe as well.

pengchengguo commented on egs/aishell2/asr1/RESULT.md:
```diff
@@ -0,0 +1,33 @@
+# Transformer result (default transformer with initial learning rate = 1.0 and epochs = 25)
```
Sorry, I missed these messages because I forgot to set a notification.
1. I received the Google drive link and upload the model.
2. The results (9.1%| 7.7%| 8.7%) are good than TDNN-Chain model (9.59%| 8.81%| 10.87%), but a little worse than another open-source Transformer implement as in https://github.com/shiyuzh2007/ASR, (8.16%| 7.64%| 8.24%).
3. The recent SOTA results on Aishell 2 corpus maybe this paper: https://arxiv.org/abs/1905.11235v2, whose results are (6.20%| 5.86%| 6.40%). It's not a Transformer framework.

pengchengguo commented on egs/aishell2/asr1/conf/decode.yaml:
```diff
@@ -0,0 +1,7 @@
+maxlenratio: 0.0
+minlenratio: 0.0
+ctc-weight: 0.5
+lm-weight: 0.7
```
I didn't tune the number of `lm-weight`. I will try to get more results with different `lm-weight`.
Ok, I will use the symlink for both training and decoding config file.

sw005320 commented:
Can you also paste the learning curve?
I'd like to check whether training is normal.
Also, please fix the CI errors in https://travis-ci.org/espnet/espnet/jobs/655109959#L2373-L2387

sw005320 commented:
Could you update RESULTS.md with your numbers? Also, please put the latest model link there (see https://github.com/espnet/espnet/blame/master/egs/aishell/asr1/RESULTS.md#L12 as a reference.).

pengchengguo commented:
1. I pasted the training details in a Google doc. Here is the link: https://docs.google.com/document/d/1EJroOUuNFQvR5wjr3cZuLhaYRVE3UdvG0K3I7VD9ypI/edit?usp=sharing
2. I also updated the results with `lm-wetight=0.5` and the model link in RESULTS.md.  
3. I fixed all Shell scripts errors checked by CI, but found another error: 'The Travis CI build failed'. The building log is at https://travis-ci.org/espnet/espnet/builds/655594369?utm_source=github_status&utm_medium=notification. I thought this problem is not caused by Aishell2 recipes.

sw005320 commented:
Final request (I forgot to mention).
Could you add a brief description of aishell2 in https://github.com/espnet/espnet/blob/master/egs/README.md ?

pengchengguo commented:
I added a description of Aishell2. I only modified the egs/README.md, but the CI test failed.

sw005320 commented:
Many thanks!
